src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/CachePoolPass.php

```diff
@@ -26,6 +26,14 @@ class CachePoolPass implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container)
     {
+        $namespaceSuffix = '';
+
+        foreach (array('name', 'root_dir', 'environment', 'debug') as $key) {
+            if ($container->hasParameter('kernel.'.$key)) {
+                $namespaceSuffix .= '.'.$container->getParameter('kernel.'.$key);
```

Comment:
:+1: for seeding cache pool namespaces with parameters from the kernel, this was missing